app/src/org/commcare/views/DrawView.java

```diff
@@ -0,0 +1,158 @@
+package org.commcare.views;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.Path;
+import android.view.Display;
+import android.view.MotionEvent;
+import android.view.View;
+import android.view.WindowManager;
+
+import org.commcare.dalvik.R;
+import org.commcare.utils.MediaUtil;
+import org.commcare.views.widgets.SignatureWidget;
+
+import java.io.File;
+
+public class DrawView extends View {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/a8c72416cc7c4632f8bf60489d4321207ff6ded4/app/src/org/commcare/views/DrawView.java#L19"></span>Custom view <code>DrawView</code> is missing constructor used by tools: <code>(Context)</code> or <code>(Context,AttributeSet)</code> or <code>(Context,AttributeSet,int)</code></td>
    </tr>
  </tbody>
</table>